google-cloud-platform/index.md

```diff
@@ -9,7 +9,7 @@ has-permalinks: false
 # {{ page.title }} <img src="images/GoogleCloudPlatform-logo.png" alt="logo for Google Cloud Platform">
 
 **Note**: There are many ways to host a Dart server. You might also be
-interested in [Heroku][] and [Dartvoid][].
+interested in [Heroku][] and [Sourcevoid](https://www.sourcevoid.com/).
```

Comment:
Also delete the Dartvoid at the end of this file (or change it and alter this link to use it).